torchvision/ops/_utils.py

```diff
@@ -61,3 +61,66 @@ def split_normalization_params(
         else:
             other_params.extend(p for p in module.parameters() if p.requires_grad)
     return norm_params, other_params
+
+
+def set_weight_decay(
+    model: torch.nn.Module,
+    weight_decay: float,
+    norm_weight_decay: Optional[float] = None,
+    norm_classes: Optional[List[type]] = None,
+    custom_keys_weight_decay: Optional[List[Tuple[str, float]]] = None,
```

Comment:
Any specific reason why we use List of tuples here instead of a `Dict[str, float]`?

Reply:
we want the order of List to define the priority